README.md

```diff
@@ -1,12 +1,39 @@
 # geocint-mapaction
 
-## Setting up AWS credentials
+This repository contains opensource geodata ETL/CI/CD pipeline developed by [Kontur](https://www.kontur.io/about/) for [MapAction](http://mapaction.org). 
+It is based on Kontur Geocint technology.
+
+## What does this pipeline do?
+
+This pipeline downloads data from various sources, including OpenStreetMap, HDX and others, and produces geospatial datasets, in form of geojsons and ESRI shapefiles, which are uploaded to S3 compatible storage and to CKAN.
+
+## Installation and cofiguration
+
+In order to make it running, you need two other repositories:
+
+* https://github.com/konturio/geocint-runner
+* https://github.com/konturio/geocint-openstreetmap
+
+For more information on geocint installation and basic configuration please see [Geocint readme](https://github.com/konturio/geocint-runner/blob/main/README.md) and [Geocint documentation](https://github.com/konturio/geocint-runner/blob/main/DOCUMENTATION.md).
+
+There are 2 things that are needed to be configured for this pipeline specifically:
+1. You need to create [Amazon S3](https://docs.aws.amazon.com/AmazonS3/latest/userguide/Welcome.html) compartible storage.
+2. You need to create you own [CKAN](https://ckan.org/) installation.
+
+For S3 storage you can use Amazon services or any other cloud/hosting provider who provides S3 compartible storage.  For S3 storage creation please refer to [Amazon S3 documentation](https://docs.aws.amazon.com/AmazonS3/latest/userguide/GetStartedWithS3.html) or documentation of your hosting provider.
+
+For CKAN installation an configuration please refer to CKAN [Maintainer’s guide](https://docs.ckan.org/en/2.9/maintaining/index.html). 
+
+The following sections describe how to specify credentials for S3 storage and CKAN for the pipeline.
+### Setting up AWS credentials
 
-Since pipeline uploads produced datasets in S3 bucket you need to make sure:
 1. `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY` and `AWS_REGION` are specified in environment variables. Please check this [manual](https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/setup-credentials.html) for more informarmation or use `aws configure`.
```

Comment:
shouldn't we also mention that AWS output format should be specified too? For example, I used json. And it started going through datasets like ourairports and etc. it fell anyway, but firstly when I didn't have output format specified, it didn't come to this step

Reply:
I do not think it's needed. it should work without it.